test/acceptance/client.js

```diff
@@ -246,7 +351,8 @@ describe('The node.js OC client', function(){
           components: [{
             name: 'hello-world',
             version: '~1.0.0'
-          }]
+          }],
+          parameters: {}
```

Comment:
uhm, why an empty `parameters` here? Just to test you can pass an empty object and that still works?